dev/fattest.simplicity/src/componenttest/topology/impl/LibertyFileManager.java

```diff
@@ -326,92 +325,110 @@ private static boolean findAny(String line, Pattern[] patterns) {
-
+    /**
+     * Create a remote file which encapsulates a machine and an absolute path.
+     * The target remote file need not exist.
+     *
+     * @param machine The machine which has the specified file.
+     * @param absPath The absolute path of the file.
+     *
+     * @return A remote file which encapsulates the machine and path.
+     */
+    public static RemoteFile createRemoteFile(Machine machine, String absPath) {
+        return new RemoteFile(machine, LibertyServerUtils.makeJavaCompatible(absPath, machine));
     }
 
-    public static void deleteLibertyDirectoryAndContents(Machine machine, String dirToDeleteAbsPath) throws Exception {
-        final String method = "deleteLibertyDirectoryAndContents";
-        Log.finer(c, method, "deleting Directory and Contents: " + dirToDeleteAbsPath);
-        try {
-            RemoteFile fileToDelete = getLibertyFile(machine, dirToDeleteAbsPath);
-            if (fileToDelete.exists()) {
-                recursivelyDeleteDirectory(machine, fileToDelete);
-            }
-        } catch (FileNotFoundException e) {
-        }
-    }
+    // Delete operations ...
 
-    private static void recursivelyDeleteDirectory(Machine machine, RemoteFile destinationToDelete) throws Exception {
-        ArrayList<String> contents = new ArrayList<String>();
-        contents = listRemoteContents(destinationToDelete);
-        for (String l : contents) {
-            RemoteFile toDelete = new RemoteFile(machine, destinationToDelete, l);
-            if (toDelete.isDirectory()) {
-                //Recurse if a directory
-                recursivelyDeleteDirectory(machine, toDelete);
+    /**
+     * Attempt to delete a file on a specified machine.
+     *
+     * Emit a log message describing one of three result cases:
+     *
+     * 1) The target file already does not exist.
+     * 2) The target file was deleted.
+     * 3) The target file was not deleted.
+     *
+     * Retry the deletion operation if the first attempt fails.
+     * See {@link RemoteFile#deleteWithRetry()}.
+     *
+     * @param machine The machine which has the specified file.
+     * @param absPath The absolute path of the file.
+     *
+     * @throws Exception Thrown in case of an error deleting the file.
+     */
+    public static void deleteLibertyFile(Machine machine, String absPath) throws Exception {
+        String method = "deleteLibertyFile";
+
+        // TODO: For now, don't log the machine, since it doesn't seem to
+        //       be significant.
+        // + " on machine '" + machine + "'";
+        String prefix = "File '" + absPath + "'";
+        Log.info(CLASS, method, prefix);
+
+        RemoteFile remoteFile = createRemoteFile(machine, absPath);
+        if ( !remoteFile.exists() ) {
+            Log.info(CLASS, method, prefix + " does not exist");
+        } else {
+            if ( remoteFile.deleteWithRetry() ) { // throws Exception
+                Log.info(CLASS, method, prefix + " was deleted");
```

Comment:
logging an INFO message on every single successful file delete is going to clutter logs too much.  Lets only log an INFO msg if something cannot be deleted.